src/sqlparser/src/ast/mod.rs

```diff
@@ -121,6 +121,7 @@ pub struct Ident {
 impl Ident {
     /// Create a new identifier with the given value and no quotes.
     /// the given value must not be a empty string.
+    // FIXME: should avoid using this function unless it's a literal or for testing.
     pub fn new_unchecked<S>(value: S) -> Self
```

Comment:
Is it possible to `pub(crate)` this?

Reply:
Unfortunately it's also used by integration tests.